src/Componenents/Map.js

```diff
@@ -0,0 +1,12 @@
+// import { Component } from 'react';
+
+// class Map extends Component {
+//   render() {
+//     return (
+
+      
+//     )
+//   }
+// }
+
+// export default Map;
```

Comment:
Feedback: Remove this file if it is not in use any longer/ the code is no longer being used 👍